web/src/containers/activity/StatePerson/StatePersonReview.js

```diff
@@ -21,8 +21,11 @@ const StatePersonReview = ({
       <div className="ds-u-margin-top--2">
         {Object.entries(years).map(([year, { amt, perc }]) => (
           <div key={year}>
-            <strong>{year} Costs:</strong> <Dollars>{amt}</Dollars> |{' '}
-            <strong>FTEs:</strong> {perc}
+            <strong>{year} </strong>
```

Comment:
I don't seem to be able to add a multiline suggestion, but I'd love to see this whole chunk wrapped in a `p` tag!